### ISSUE TITLE
fnft_nsev_inverse fails if D is not a power of 2

### ISSUE THREAD

**Issue by simgunz** [FastNFT/FNFT#39]
Is this a bug or the intended behavior? In the second case it would be good to add this fact to the documentation.

Run:
`q = mex_fnft_nsev_inverse([], [-1 1], [1i*0.3], [1i], 1023, [-1 1], 1 )`

Error:
```
FNFT Error: Invalid argument D.
 in fnft_nsev_inverse(148)-0.2.1
```

**Comment by wahls** [comment]
Thank you for the report! 

It is intended, the number of samples in fnft_nsev_inverse currently has to be a power of two. The documentation will be updated at the next occasion. 

I am leaving this open for now as a reminder.

**Comment by simgunz** [comment]
Ok. Thanks for the reply.

**Comment by wahls** [comment]
Fixed in v0.2.2.
